src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -58,8 +58,9 @@ public ObservedTagSet(string tags)
         private readonly object _lock = new object();
         private readonly Dictionary<string, ObservedProvider> _providers = new Dictionary<string, ObservedProvider>(); // Tracks observed providers and counters.
         private const int Indent = 4; // Counter name indent size.
-        private int _maxNameLength = 40; // Allow room for 40 character counter names by default.
+        private const int CounterValueLength = 15;
 
+        private int _maxNameLength = 60; // Allow room for 60 character counter names by default.
```

Comment:
I assume this default is never used because we always calculate a new value for the field in AssignRowsAndInitializeDisplay() before using it? If so I'd recommend letting this have the default value 0. Assigning 60 and the comment gives the impression this value we pick here matters.

Reply:
I think this value mattered with the old implementation of using a ternary operator. Now, it can start at 0 as you stated.

Reply:
I think this value mattered with the old implementation of using a ternary operator. Now, it can start at 0 as you stated.